### PR TITLE
feat: add crane and crane-runner

### PR DESCRIPTION
**Pull request by djzager**
Adding crane and crane-runner together as crane-runner requires the binary built in the crane image.

### CODE REVIEW

File: ci-operator/config/konveyor/crane-runner/konveyor-crane-runner-main.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: Dockerfile
+  from: ubi
+  inputs:
```
Comment (djzager) [inline ci-operator/config/konveyor/crane-runner/konveyor-crane-runner-main.yaml:18]: Skipping for now adding `crane` container image as an input for the sake of getting into openshift/release. That'll be taken care of in a follow-up PR.

**Review by droslean — APPROVED** [review]

**Pull request merged**